### PR TITLE
Fix a lot content_unit server calls from playlist

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Pages/PlaylistCollection/Container.js [hunk 1]
```diff
@@ -45,14 +45,16 @@ const PlaylistCollectionContainer = ({ cId, t, cuId }) => {
...
-            const cu = content_units.find(x => x.id === cuID);
-            if (!cu || !cu.files || !cu.tags || !cu.sources) {
-              dispatch(actions.fetchUnit(cuID));
-            }
-          }
+        const cusForFetch = cuIDs.filter(cuID => {
+          if (fullUnitFetchedMap[cuID] || wipMap.units[cuID] || errorMap.units[cuID])
+            return false;
+          const cu = content_units.find(x => x.id === cuID);
+          return !cu?.files;
```
Comment (davgur) [inline src/components/Pages/PlaylistCollection/Container.js:52]: @bbfsdev I remove check tags and sources from here, look like its not need

Comment (bbfsdev) [reply src/components/Pages/PlaylistCollection/Container.js:52]: That is not true it solves this issue:
https://issues.kbb1.com/issue/AR-874

Does this causes the problem?

File: src/redux/modules/mdb.js [hunk 2]
```diff
@@ -159,13 +171,17 @@ const setStatus = (state, action) => {
...
 
   switch (action.type) {
     case FETCH_UNIT:
-      wip.units = { ...wip.units, [action.payload]: true };
+      wip.units     = { ...wip.units, [action.payload]: true };
       fetched.units = { ...fetched.units, [action.payload]: true };
       break;
+    case FETCH_UNITS_BY_IDS:
+      units.wip = action.payload.id?.reduce((acc, id) => ({ ...acc, [id]: true }), {});
+      wip.units = { ...wip.units, ...units.wip };
```
Comment (bbfsdev) [inline src/redux/modules/mdb.js:183]: This should be a full fetch, right? in that case, in that case also update 

fetched.units = { ...fetched,units, all the new uids that you are going to load... }

see line 179

File: src/components/Pages/PlaylistCollection/Container.js [hunk 3]
```diff
@@ -45,14 +45,16 @@ const PlaylistCollectionContainer = ({ cId, t, cuId }) => {
...
         });
+
+        if (cusForFetch?.length > 0) {
+          dispatch(actions.fetchUnitsByIDs({ id: cusForFetch, with_tags: true, with_files: true }));
```
Comment (bbfsdev) [inline src/components/Pages/PlaylistCollection/Container.js:56]: If this is a full fetch, you need to fetch also with_sources

**Review by bbfsdev — CHANGES_REQUESTED** [review]

File: src/components/Pages/PlaylistCollection/Container.js (hunk 3 above)
Comment (davgur) [reply src/components/Pages/PlaylistCollection/Container.js:56]: but here I dont need sources, I think that can remove tags too. What you think?

File: src/components/Pages/PlaylistCollection/Container.js (hunk 1 above)
Comment (davgur) [reply src/components/Pages/PlaylistCollection/Container.js:52]: is issue is about played now CU right? So why you need fetch data for all units of collection?

File: src/redux/modules/mdb.js (hunk 2 above)
Comment (davgur) [reply src/redux/modules/mdb.js:183]: no, I fetch with params, so it can be not full (on this place I need only files)

**Review by bbfsdev — APPROVED** [review]

**Pull request merged**